pkg/pillar/cmd/zedagent/handlecertconfig.go

```diff
@@ -36,22 +37,22 @@ func parseControllerCerts(ctx *zedagentContext, contents []byte) {
 	for _, cfgCert := range cfgCerts {
 		computeConfigElementSha(h, cfgCert)
 	}
-	newConfigHash := h.Sum(nil)
-	if bytes.Equal(newConfigHash, certConfigHash) {
+	newHash := h.Sum(nil)
+	if bytes.Equal(newHash, certHash) {
 		return
 	}
 	log.Infof("parseControllerCerts: Applying updated config\n"+
 		"Last Sha: % x\n"+
 		"New  Sha: % x\n"+
 		"cfgCertList: %v\n",
-		certConfigHash, newConfigHash, cfgCerts)
+		certHash, newHash, cfgCerts)
```

Comment:
This isn't new, but the %v means the log message are kilobytes in size (depends how many certs there are) and will probably be dropped. logging the number of certs we received but not their content might make sense.
Don't know if there are other places we log the cert content but it isn't useful

Reply:
Yeah, it completely makes sense to me. Logging certs bytes in logs is not useful. We are logging certs at many places, it should be replaced by the number of certs as suggested. Should I do it in the same PR or can I take it as a separate clean up?

Reply:
Can do that cleanup in a separate PR.